e107_plugins/social/e_comment.php

```diff
@@ -34,7 +34,7 @@ public function config() // Admin Area Configuration.
 
 		if($engine == 'social::facebook' && empty($this->facebookActive))
 		{
-			e107::getMessage()->addInfo("Facebook comments requires that you have a facebook App ID. See the 'social login' area in admin-preferences to add one.");
+			e107::getMessage()->addInfo(LAN_SOCIAL_WARNING);
```

Comment:
Wor this in English_global.php 

![soc2](https://cloud.githubusercontent.com/assets/17430115/22558333/9817d1ee-e96d-11e6-8bf7-c06ca4bb0e1b.png)